containers/ddev-webserver/Dockerfile

```diff
@@ -262,7 +263,9 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -qq install -o Dpkg::Options::="--for
     rsync \
     sqlite3 \
     unzip \
+    vim-tiny \
```

Comment:
vim-tiny is already explicitly added upstream.

```suggestion
```

Reply:
This part of code is related to the hardened images. I checked it, and there is no `vi` or `vim` in the hardened image.